modules/cnf-topology-aware-lifecycle-manager-apply-policies.adoc

```diff
@@ -14,6 +14,7 @@ You can update your managed clusters by applying your policies.
 * Provision one or more managed clusters.
 * Log in as a user with `cluster-admin` privileges.
 * Create {rh-rhacm} policies in the hub cluster.
+* {rh-rhacm} version 2.9 or later.
```

Comment:
This is a general requirement for TALM 4.16, not particular to this section. Is there a better place for this requirement?

Reply:
Added this info as a prerequisite to the installing sections also.